website/docs/en/config/externals.mdx

```diff
@@ -125,10 +125,10 @@ module.exports = {
   // or
   externals: {
     lodash: {
-      root: '_', // required, indicates global variable
-      commonjs: 'lodash', // required
-      commonjs2: 'lodash', // required
-      amd: 'lodash', // optional
+      root: '_', // indicates global variable
```

Comment:
Should we note that these properties are all required when libraryTarget is umd?